### ISSUE TITLE
[BUG] No Callbacks Received from Registered Tournament Provider

### ISSUE THREAD

**Issue by iDanielWongi** [RiotGames/developer-relations#944]
**Bug Description**  
The callback from the League of Legends server to the registered URL is not functioning as expected. Despite the server receiving the request when using postman, no proper acknowledgment or expected behavior occurs from riot - dont see any calls.

**Problem Description**  
There is no callback from Riot that the game is over. And the correct ports are activated, I try to do it through postman and it works as it should.

**Expected Result**  
After the game completes, the server should receive a callback with the game results. This should trigger the server to process the data, update the database, and perform necessary actions based on the game results. The server should acknowledge the callback with a 200 OK response.

**Actual Result**  
Nothing happens!

I haven't received any requests on the url I used when registering a tournament provider from the tournament codes generated with that provider id

**Comment by tisbells** [comment]
Hi there, we are looking into this and will post back when we have more info.

**Comment by Dallin-Cawley** [comment]
From the future, has this been resolved? I am also not receiving any callbacks.

**Comment by tisbells** [comment]
Could those experiencing this issue please send the following to our [support site](https://support-developer.riotgames.com/):

Recent tournament codes
The provider URL
Confirmation of whether the game looks as expected using [Get Game](https://developer.riotgames.com/apis#tournament-v5/GET_getGames)

**Comment by ruuffian** [comment]
I've been experiencing the same thing- I think this issue has something to do with SSL certs from CA's that are too new. Is there a list of what places to buy/create SSL certs that Riot will accept? I don't want to spend money on certs that don't fix the problem.

**Comment by Lakuna** [comment]
> I've been experiencing the same thing- I think this issue has something to do with SSL certs from CA's that are too new. Is there a list of what places to buy/create SSL certs that Riot will accept? I don't want to spend money on certs that don't fix the problem.

From [the API documentation page](https://developer.riotgames.com/docs/lol#tournament-api_server-callback):

![Image](https://github.com/user-attachments/assets/4f02600d-d5ea-4c52-9170-1c216b24cef7)

When I asked about this in the developer Discord they suggested I just don't use SSL at all for best results.

**Comment by ruuffian** [comment]
> When I asked about this in the developer Discord they suggested I just don't use SSL at all for best results.

I opened a support ticket with Riot >1 year ago and they specifically told me to never use http, only use https or it won't work. The ticket was closed with no further comment. @Lakuna Do you have tournament callbacks working with HTTP???? That is so, so frustrating if you do. I keep being told incorrect information in Riot support tickets.

**Comment by Lakuna** [comment]
> > When I asked about this in the developer Discord they suggested I just don't use SSL at all for best results.
> 
> I opened a support ticket with Riot >1 year ago and they specifically told me to never use http, only use https or it won't work. The ticket was closed with no further comment. [@Lakuna](https://github.com/Lakuna) Do you have tournament callbacks working with HTTP???? That is so, so frustrating if you do. I keep being told incorrect information in Riot support tickets.

No, I couldn't find a good serverless host that would allow me to use HTTP so I just settled for having my users submit game IDs after the games. I asked another friend who was working on a different project and he apparently got it working with an AWS Lambda function though.
